newsrevenuehub/rev-engine
# combine NON_DONATION_PAGE_SUBDOMAINS and ORG_PORTAL_SUBDOMAINS into  DASHBOARD_SUBDOMAINS

#### What's this PR do?

- creates a new environment variable: `DASHBOARD_SUBDOMAINS` 
- removes/deprecates two others: `NON_DONATION_PAGE_SUBDOMAINS` and `ORG_PORTAL_SUBDOMAINS`

#### Why are we doing this? How does it help us?

I believe the two variables were serving the same purpose and trying to handle them both was making the unnecessarily complex. This removes those and creates a new one that is hopefully more clear in the name.

#### How should this be manually tested?

- make sure you can access the dashboard at https://dev-1600.revengine-review.org/
- set the `DASHBOARD_SUBDOMAINS` Config Var to `foo:bar` 
- confirm that you can access the dashboard at:
   - https://foo.revengine-review.org/
   - https://bar.revengine-review.org/
- remove the `DASHBOARD_SUBDDOMAINS` Config Var
- confirm you can access it through the defaults:
   - https://support.revengine-review.org/
   - https://www.revengine-review.org/
   - https://dashboard.revengine-review.org/

#### Have automated unit tests been added?

Existing unit tests cover these changes.

#### How should this change be communicated to end users?

N/A

#### Are there any smells or added technical debt to note?

No.

#### Has this been documented? If so, where?

No.

#### What are the relevant tickets?
**(Note: Please use syntax [JIRA-123] to link any relevant tickets)**
[DEV-1600]

#### Do any changes need to be made before deployment to production (adding environment variables, for example)?

No, but later we may want to set the `DASHBOARD_SUBDOMAINS` variable.

#### Are there next steps? If so, what? Have tickets been opened for them?

No.
